domain/src/main/java/org/oppia/android/domain/topic/TopicController.kt

```diff
@@ -143,6 +144,30 @@ class TopicController @Inject constructor(
     )
   }
 
+  /**
+   * Fetches a chapter given a topic ID, story ID and exploration ID.
+   *
+   * @param topicId the ID corresponding to the topic which contains this story
+   * @param storyId the ID corresponding to the story which needs to be returned
+   * @param explorationId the ID corresponding to the exploration which needs to be returned
+   * @return a [DataProvider] for [ChapterSummary]
+   */
+  fun getChapter(
```

Comment:
Maybe instead: retrieveChapter? The helper could be called 'fetchChapter' since we generally call data structure retrievers at that logical level 'fetchers'.

At this level, 'get' usually means a trivial fetch vs. a more expensive operation to retrieve something.

Reply:
Also: please add tests for this new method.